### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by ahie**
This commit does not include refactoring the Components folder

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+dist: trusty
+sudo: false
+
+language: node_js
+node_js:
+  - '10'
```
Comment (charn) [inline .travis.yml]: Should we use node 12 here i.e. the same as in the Dockerfile?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+dist: trusty
```
Comment (charn) [inline .travis.yml]: `xenial` is the Travis default. Do we need to define `trusty` here`

File: .travis.yml (hunk 1 above)
Comment (ahie) [reply .travis.yml]: Bumped version up to 12.

File: .travis.yml (hunk 2 above)
Comment (ahie) [reply .travis.yml]: Removed `dist`.

**Pull request merged**